main.go

```diff
@@ -47,79 +51,21 @@ func newFn() *cli.App {
 	// AppHelpTemplate is the text template for the Default help topic.
-	// cli.go uses text/template to render templates. You can
-	// render custom help text by setting this variable.
-	cli.AppHelpTemplate = `
-	{{if .ArgsUsage}}{{else}}{{.Description}} - Version {{.Version}} 
-
-	ENVIRONMENT VARIABLES:
-	   FN_API_URL   Fn server address
-	   FN_REGISTRY  Docker registry to push images to, use username only to push to Docker Hub - [[registry.hub.docker.com/]USERNAME]{{end}}{{if .VisibleCommands}}
-
-	{{if .ArgsUsage}}{{else}}GENERAL COMMANDS:{{end}}{{end}}{{range .VisibleCategories}}{{if .Name}}
-
-	{{.Name}}:{{end}}{{range .VisibleCommands}}
-		{{join .Names ", "}}{{"\t"}}{{.Usage}}{{end}}{{end}}{{if .VisibleFlags}}
-
-	GLOBAL OPTIONS:
-	   {{range $index, $option := .VisibleFlags}}{{if $index}}
-	   {{end}}{{$option}}{{end}}{{end}}
-
-	FURTHER HELP:
-	   See 'fn <command> --help' for more information about a command.
-
-	LEARN MORE:
-	   https://github.com/fnproject/fn
+	// cli.go uses text/template to render templates. You can render custom help text by setting this variable.
+	cli.AppHelpTemplate = `{{"\n"}}{{"\t"}}{{if not .ArgsUsage}}{{boldred .Description}}{{"\t"}}{{boldred "-"}}{{"\t"}}{{boldred "Version "}}{{boldred .Version}}{{"\n"}}{{"\n"}}{{"\t"}}{{bold "ENVIRONMENT VARIABLES"}}{{"\n"}}{{"\t"}}{{"\t"}}FN_API_URL{{"\t"}}{{italic "Fn server address"}}{{"\n"}}{{"\t"}}{{"\t"}}FN_REGISTRY{{"\t"}}{{italic "Docker registry to push images to, use username only to push to Docker Hub - [[registry.hub.docker.com/]USERNAME]"}}{{if .VisibleCommands}}{{"\n"}}{{"\n"}}{{"\t"}}{{bold "GENERAL COMMANDS"}}{{end}}{{else}}{{range .VisibleCategories}}{{if .Name}}{{bold .Name}}{{end}}{{end}}{{"\n"}}{{"\t"}}{{"\t"}}{{boldcyan .HelpName}}{{if .Usage}}{{" - "}}{{italic .Usage}}{{"\n"}}{{"\n"}}{{"\t"}}{{bold "USAGE"}}{{"\n"}}{{"\t"}}{{"\t"}}{{boldcyan .HelpName}}{{if .VisibleFlags}}{{cyan " [global options]"}}{{end}} {{if .ArgsUsage}}{{brightred .ArgsUsage}}{{end}}{{if .Flags}}{{yellow " [command options]"}}{{end}}{{if .Description}}{{"\n"}}{{"\n"}}{{"\t"}}{{bold "DESCRIPTION"}}{{"\n"}}{{"\t"}}{{"\t"}}{{.Description}}{{end}}{{end}}{{end}}{{range .VisibleCategories}}{{if .Name}}{{"\n"}}{{"\n"}}{{"\t"}}{{bold .Name}}{{end}}{{range .VisibleCommands}}{{"\n"}}{{"\t"}}{{"\t"}}{{join .Names ", "}}{{"\t"}}{{"\t"}}{{"\t"}}{{.Usage}}{{end}}{{end}}{{if .VisibleFlags}}{{"\n"}}{{"\n"}}{{"\t"}}{{if not .ArgsUsage}}{{bold "GLOBAL OPTIONS"}}{{else}}{{bold "COMMAND OPTIONS"}}{{end}}{{"\n"}}{{"\t"}}{{"\t"}}{{range $index, $option := .VisibleFlags}}{{if $index}}{{"\n"}}{{"\t"}}{{"\t"}}{{end}}{{$option}}{{end}}{{end}}{{"\n"}}{{"\n"}}{{"\t"}}{{bold "FURTHER HELP:"}}{{"\t"}}{{italic "See "}}{{"'"}}{{brightcyan .HelpName}}{{brightcyan " <command> --help"}}{{"'"}}{{italic " for more information about a command."}}{{if not .ArgsUsage}}{{"\n"}}{{"\n"}}{{"\t"}}{{bold "LEARN MORE:"}}{{"\t"}}{{"\t"}}{{underlinebrightred "https://github.com/fnproject/fn"}}{{else}}{{end}}
```

Comment:
I don't think you need template commands like : `{{"\n"}}{{"\n"}}{{"\t"}}` in this template - normally you'd just include them as text in the raw template.